package.json

```diff
@@ -16,7 +16,7 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC",
+  "license": "apache-2.0",
```

Comment:
https://spdx.org/licenses/Apache-2.0.html

--- 

SPDX short identifier is `Apache-2.0` instead of `apache-2.0`. I don't think it's necessary to update (probably works in an incase-sensitive way) but something nice to know for future ;)